packages/block-library/src/latest-posts/edit.js

```diff
@@ -34,13 +34,17 @@ import { withSelect } from '@wordpress/data';
 const CATEGORIES_LIST_QUERY = {
 	per_page: -1,
 };
+const USERS_LIST_QUERY = {
+	per_page: -1,
```

Comment:
Wonder if this it going to scale okay? Just trying to think what'll happen on a site with hundreds of thousands or millions of users.

Reply:
Oh, never mind me just read the REST pagination docs.
> Large queries can hurt site performance, so per_page is capped at 100 records. If you wish to retrieve more than 100 records, for example to build a client-side list of all available categories, you may make multiple API requests and combine the results within your application.
https://developer.wordpress.org/rest-api/using-the-rest-api/pagination/